packages/grant-explorer/src/features/api/utils.ts

```diff
@@ -369,6 +381,54 @@ const POLYGON_MUMBAI_TOKENS: VotingToken[] = [
   },
 ];
 
+const AVALANCHE_TOKENS: VotingToken[] = [
+  {
+    name: "AVAX",
+    chainId: ChainId.AVALANCHE,
+    address: ethers.constants.AddressZero,
+    decimal: 18,
+    logo: TokenNamesAndLogos["AVAX"],
+    redstoneTokenId: RedstoneTokenIds["AVAX"],
+    defaultForVoting: true,
+    canVote: true,
+  },
+  {
+    name: "USDC",
+    chainId: ChainId.AVALANCHE,
+    address: "0xB97EF9Ef8734C71904D8002F8b6Bc66Dd9c48a6E",
+    decimal: 6,
+    logo: TokenNamesAndLogos["USDC"],
+    redstoneTokenId: RedstoneTokenIds["USDC"],
+    defaultForVoting: false,
+    canVote: true,
+    permitVersion: "2",
+  },
```

Comment:
To confirm, for Avalanche we support AVAX and USDC? The [ticket](https://github.com/gitcoinco/grants-stack/issues/2308) mentions AVAX, ETH, wETH, and DAI.